src/types/messages.ts

```diff
@@ -68,6 +68,16 @@ export interface HasAppModuleMessageWhereConditions {
   operator: string;
   value: string;
 }
+export interface HasTagWhereHasConditions {
```

Comment:
we already have a declared interface that you can also use for this, it's called **WhereCondition**.